files/default/supervisor-solr.conf

```diff
@@ -20,7 +20,7 @@ numprocs=1
 process_name=%(program_name)s-%(process_num)02d
 
 ; Log files.
-stdout_logfile=/var/log/solr/stdout.log
+stdout_logfile=/var/log/solr/solr.log
 stderr_logfile=/var/log/solr/stderr.log
```

Comment:
would we not look at borrowing from upstream?
https://issues.apache.org/jira/browse/SOLR-14410
https://issues.apache.org/jira/secure/attachment/13000189/solr.service

https://github.com/apache/solr/pull/428/files#diff-f960855d84316fcf7d8cbbf25c2bd61d1493d66508e7a5f0331ed018e592fb3c